com.unity.render-pipelines.universal/Documentation~/renderer-feature-decal.md

```diff
@@ -1,51 +1,132 @@
+
+1. [Add the Decal Renderer Feature](urp-renderer-feature-how-to-add.md) to the URP Renderer.
+
+2. Create a Material, and assign it the `Shader Graphs/Decal` shader. In the Material, select the Base Map and the Normal Map.
+
+    ![Example decal Material](Images/decal/decal-example-material.png)
+
+3. Create a new Decal Projector GameObject, or add a [Decal Projector component](#decal-projector-component) to an existing GameObject.
+
+The following illustration shows a Decal Projector in the Scene.
+
+![Decal Projector in the Scene.](Images/decal/decal-projector-selected-with-inspector.png)
+
+For more information, see also [Decal Projector component](#decal-projector-component).
+
+## Limitations
+
+This feature has the following limitations:
+
+* The decal projection does not work on transparent surfaces.
 
 ## Decal Renderer Feature properties
 
 This section describes the properties of the Decal Renderer Feature.
 
-### Techniques
+![Decal Renderer Feature, Inspector view.](Images/decal/decal-rf-inspector.png)<br/>*Decal Renderer Feature, Inspector view.*
+
+### Technique
+
+Select the rendering technique for the Renderer Feature.
 
-Select the rendering techniques that suits your project.
+This section describes the options in this property.
 
 #### Automatic
 
-Unity selects the technique automatically based on the build platform.
+Unity selects the rendering technique automatically based on the build platform.
 
 #### DBuffer
 
-Renders decals into DBuffer and then applied during opaque rendering. Requires DepthNormal prepass which makes not viable solution for tiled based renderers.
-Does not work on particles and terrain details.
+Unity renders decals into the Decal buffer (DBuffer). Unity overlays the content of the DBuffer on top of the opaque objects during the opaque rendering.
+
+Selecting this technique reveals the **Surface Data** property. The Surface Data property lets you specify which surface properties of decals Unity blends with the underlying meshes. The Surface Data property has the following options:
+
+* **Albedo**: decals affect the base color and the emission color.
+* **Albedo Normal**: decals affect the base color, the emission color, and the normals.
+* **Albedo Normal MAOS**: decals affect the base color, the emission color, the normals, the metallic values, the smoothness values, and the ambient occlusion values.
 
-|__Propery__           | __Description__ |
-| -------------------- |---------------- |
-| __Surface Data__     | Allows specifying which decals surface data should be blended with surfaces. |
-| _Albedo_             | Decals will affect only base color and emission.|
-| _Albedo Normal_      | Decals will affect only base color, normal and emission|
-| _Albedo Normal MAOS_ | Decals will affect only base color, normal, metallic, ambient occlusion, smoothness and emission.|
+**Limitations:**
 
+* This technique requires the DepthNormal prepass, which does not work with tile-based rendering.
```

Comment:
I feel like this sentance refers that technique does no work at all with tile based render. It does work, it just not effective as DepthNormal prepass in tile-based rendering only adds cost (Where in dekstop it can actually increase performance by a bit)

Reply:
Thanks!
How about this:
***
* This technique requires the DepthNormal prepass, which makes the technique less efficient on GPUs that implement tile-based rendering.
***
And if this version is correct, can we add a short sentence about why it's less efficient (or just inefficient)?

Reply:
Yeap looks good, I think this info should be in depth prepass docs if we have them